community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/scripts/util.py

```diff
@@ -1984,15 +1984,18 @@ def template_info(self, template_link):
         # del template.metadata
 
         # translate gpus into an easier-to-read format
+        gpu_prefix = "nvidia-"
```

Comment:
When type is not specified then CTK uses [this definition](https://github.com/GoogleCloudPlatform/cluster-toolkit/blob/26f95d86f36ddace4fedb76f25e6bc5b25c403e7/modules/internal/gpu-definition/main.tf#L48-L74) to populate. This prefix seems compatible with all these types.

But I have also seen `type = "NVIDIA H200 141GB"` used and accepted. 

I think it would be safer to do a lower(type).contains("nvidia")
(or w/e correct syntax is)